src/main/resources/application.properties

```diff
@@ -491,6 +491,19 @@ kafka.streams.transient.processing.exception.threshold.interval=PT30M
 # Refer to https://kafka.apache.org/documentation/#consumerconfigs for available options.
 # alpine.kafka.processor.<name>.consumer.<consumer.config.name>=
 
+# Required
+# Configures the Kafka processor responsible for ingesting mirrored vulnerability
```

Comment:
should we also add these properties to readme since they are configurable by user?

Reply:
Good point! I'm looking into generating documentation based on contents of `application.properties` (https://github.com/DependencyTrack/hyades/issues/1154). I would consider this to be a side effect of that effort.